template/src/subComponents/CopyJoinInfo.tsx

```diff
@@ -36,39 +39,52 @@ const SHARE = gql`
   }
 `;
 
-const ParticipantView = (props: {showText?: boolean}) => {
+const CopyJoinInfo = (props: {showText?: boolean}) => {
   const {phrase} = useParams<{phrase: string}>();
   const {data, loading, error} = useQuery(SHARE, {
     variables: {passphrase: phrase},
   });
+  const copiedToClipboardText = useString(
+    'copiedToClipboardNotificationLabel',
+  )();
+  const meetingInviteText =
+    useString<MeetingInviteInterface>('meetingInviteText');
   const copyToClipboard = () => {
-    Toast.show({text1: 'Copied to Clipboard', visibilityTime: 1000});
+    Toast.show({text1: copiedToClipboardText, visibilityTime: 1000});
     if (data && !loading) {
-      let stringToCopy = '';
-      if ($config.FRONTEND_ENDPOINT) {
-        stringToCopy += `Meeting - ${data.share.title}\nURL for Attendee: ${$config.FRONTEND_ENDPOINT}/${data.share.passphrase.view}`;
-        if (data.share.passphrase.host) {
-          stringToCopy += `\nURL for Host: ${$config.FRONTEND_ENDPOINT}/${data.share.passphrase.host}`;
-        }
-      } else {
-        if (platform === 'web') {
-          stringToCopy += `Meeting - ${data.share.title}\nURL for Attendee: ${window.location.origin}/${data.share.passphrase.view}`;
-          if (data.share.passphrase.host) {
-            stringToCopy += `\nURL for Host: ${window.location.origin}/${data.share.passphrase.host}`;
-          }
-        } else {
-          stringToCopy += `Meeting - ${data.share.title}\nAttendee Meeting ID: ${data.share.passphrase.view}`;
-          if (data.share.passphrase.host) {
-            stringToCopy += `\nHost Meeting ID: ${data.share.passphrase.host}`;
-          }
-        }
-      }
-      if (data.share.pstn) {
-        stringToCopy += `\nPSTN Number: ${data.share.pstn.number}\nPSTN Pin: ${data.share.pstn.dtmf}`;
-      }
+      let baseURL =
+        platform === 'web'
+          ? $config.FRONTEND_ENDPOINT || window.location.origin
+          : null;
+      let stringToCopy = meetingInviteText({
+        meetingName: data.share.title,
+        url: baseURL
+          ? data.share.passphrase.host
```

Comment:
Copy ones are a little painful to read everywhere, let's put then and organise them in one.